tests/subsys/edac/ibecc/src/ibecc.c

```diff
@@ -336,15 +334,10 @@ void test_main(void)
 		k_oops();
 	}
 #endif
-
-	ztest_test_suite(ibecc,
-			 ztest_unit_test(test_trigger_nmi_handler),
-			 ztest_unit_test(test_ibecc_initialized),
-			 ztest_unit_test(test_ibecc_api),
-			 ztest_unit_test(test_edac_dummy_api),
-			 ztest_unit_test(test_ibecc_error_inject_api),
-			 ztest_unit_test(test_ibecc_error_inject_test_cor),
-			 ztest_unit_test(test_ibecc_error_inject_test_uc)
-			);
-	ztest_run_test_suite(ibecc);
+	trigger_nmi_handler();
+	ibecc_initialized();
+	ibecc_api();
```

Comment:
why do you move some tests to setup function? Probably only `ibecc_initialized` can be moved here.

Reply:
A better method will correct this. Move all sequenced testing into one testing.